lo_2010.cpp

```diff
@@ -0,0 +1,51 @@
+// details:
+// from lo et al 2010 - Population pharmacokinetics of vancomycin in premature Malaysian neonates:
+// identification of predictors for dosing determination.
+
+[PARAM] @annotated
+CL  : 0.345 : Clearance (L/hr)
+V   : 1.75  : Volume (L) 
+wt  : 2.9   : Weight (kg)
+pma : 34.8  : post-menstral age (wk)
+scr : 0.9   : serum creatinine (mg/dL)
+Fsga :   ?  : small-for-gestational-age
+spiro :   ? : spironolactone (L/kg)
+
+
+[CMT] @annotated
+CENT : Central compartment (mg)
+
+[PKMODEL]
+ncmt=1, trans=11
+
+[MAIN]
+D_CENT = 1; 
+double NORM_WT = 70;
+double NORM_PMA = 30;
+double Fsga = ?;
```

Comment:
this is how you define values inside your code - you have already set these as parameters so you don't need to re-define them